extensions/roc-plugin-style-sass/src/index.js

```diff
@@ -32,11 +32,16 @@ export const roc = {
         extension: 'roc-plugin-style-css',
         hook: 'add-style',
         description: 'Adds Sass support to Webpack.',
-        action: ({ context: { config: { settings } } }) => () => () => ({
-            extensions: ['sass', 'scss'],
-            loaders: `${require.resolve('sass-loader')}?${settings.build.sass.useBourbon ?
-                `includePaths[]=${BOURBON_PATH}` : ''
-            }`,
-        }),
+        action: ({ context: { config: { settings } } }) => () => () => {
+            const params = [
+                settings.build.sass.useBourbon ? `includePaths[]=${BOURBON_PATH}` : '',
+                settings.build.style.sourceMap ? 'sourceMap' : '',
+            ].filter(v => v !== '');
```

Comment:
We do not need to do this check here since the empty string is falsy by default.